infrasim/console.py

```diff
@@ -88,11 +88,11 @@ def run(self):
                 continue
 
 
-def _start_console():
+def _start_console(instance="default"):
```

Comment:
It comes to me that `instance` can be a environment variable, so you dont' have to pass it to these functions.